File: src/main/java/com/hmo/cyclism/repository/CyclistRepository.java
package com.hmo.cyclism.repository;

import com.hmo.cyclism.model.Cyclist;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CyclistRepository extends ReactiveMongoRepository<Cyclist, String> {


}

File: src/main/kotlin/com/hmo/cyclism/repository/CyclistRepository.kt
package com.hmo.cyclism.repository

import com.hmo.cyclism.model.Cyclist
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface CyclistRepository : CrudRepository<Cyclist, String> {

}